src/ImGui.NET.SampleProgram/Program.cs

```diff
@@ -83,6 +83,12 @@ static void Main(string[] args)
 
         private static unsafe void SubmitUI()
         {
+            ImGui.Text("String!");
+            ImGui.Text("String!\0");
+            ImGui.Text("temp".AsSpan());
+            ImGui.Text(null);
```

Comment:
let me check if these work in 1.89.5 (before your PR was merged).

Reply:
I tested it and `null` doesn't work on 1.89.5 either. `string.Empty` does though. I'm currently looking into it.

Reply:
its failing over here, looks like imgui itself doesn't allow null. so we can skip that test-case.
![image](https://github.com/ImGuiNET/ImGui.NET/assets/5079646/b61813cb-e52f-4935-8143-4417638466b1)